docs/guide/basics/sizing.md

```diff
@@ -0,0 +1,86 @@
+<script setup>
+import { Sandpack } from 'sandpack-vue3'
+import sandpackTheme from '../../.vitepress/theme/sandpackTheme.json'
+import sizeIconExample from './examples/size-icon-example/files.ts'
+import sizeIconCssExample from './examples/size-icon-css-example/files.ts'
+import sizeIconFontExample from './examples/size-icon-font-example/files.ts'
+import sizeIconTailwind from './examples/size-icon-tailwind-example/files.ts'
+</script>
+
+# Sizing
+
+By default the size of the icons are `24px` . This is adjustable by using passing the `size` prop or change it by using CSS.
+
+## Adjusting size with `size` prop
+
+<Sandpack
+  template="react"
+  :theme="sandpackTheme"
+  :files="sizeIconExample"
+  :customSetup='{
+    dependencies: {
+      "lucide-react": "latest"
+    }
+  }'
+  :options="{
+    editorHeight: 300,
+  }"
+/>
+
+## Adjusting size with CSS
+
+CSS properties `width` and `height` can be used to adjust icon size.
+
+<Sandpack
+  template="react"
+  :theme="sandpackTheme"
+  :files="sizeIconCssExample"
+  :customSetup='{
+    dependencies: {
+      "lucide-react": "latest"
+    }
+  }'
+  :options="{
+    editorHeight: 300,
+  }"
+/>
+
+
+### Resize based on font size
```

Comment:
```suggestion
### Dynamically change icon size based on font size
```

Reply:
Don't we need the in front of size and font size?
```diff
- ### Dynamically change icon size based on font size
+ ### Dynamically change the icon size based on the font size
```